src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  // Remove tab characters and split the string into an array
+  const styleStringArray = sourceString.replaceAll('\t', '').split(';');
```

Comment:
tab only?
I think this is redundant.
It should work without the `replaceAll` method, since u use `trim` later